README.md

```diff
@@ -686,7 +686,7 @@ The OpenTelemetry Operator *might* work on versions outside of the given range,
 
 | OpenTelemetry Operator | Kubernetes           | Cert-Manager        |
 |------------------------|----------------------|---------------------|
-| v0.90.0                | v1.23 to v1.28       | v1                  |
+| v0.90.0                | v1.23 to v1.29       | v1                  |
```

Comment:
this should be bumped in the next release, we didn't merge support for 1.29 in the 0.90 release